schemas/theme/section_schema.json

```diff
@@ -2,6 +2,7 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "title": "Shopify Liquid Theme Section Schema",
   "type": "object",
+  "additionalProperties": false,
```

Comment:
Would this be risky to enforce no random schema values at the section schema root?

Technically our [documentation does not state outright that we accept any other values than whats documented](https://shopify.dev/docs/themes/architecture/sections/section-schema#content).

Reply:
I believe it's a good idea. While the documentation doesn't enforces it, the Assets API (used by the CLI and the code editor) doesn't allow extra attributes, so it seems like a nice addition :)